docs/api/commands/getallcookies.mdx

```diff
@@ -60,7 +60,7 @@ redirects back to our site, which sets a session cookie.
 
 ```javascript
 cy.contains('Log in').click()
-cy.origin('https://login.site.com', () => {
+cy.origin('https://cypress.io', () => {
```

Comment:
If the goal is to recommend only visiting sites under one's control, should we use localhost examples instead? Users don't control the cypress.io site, so maybe it's not sending the right message to use it in the examples?

Reply:
I did think about this. I thought maybe the examples would be seen as ourselves, like Cypress tests to test Cypress which we control? Localhost presented some issues with some examples because we were explicitly mentioning different origins.

Maybe some more examples could be localhost.